ovos_PHAL_plugin_homeassistant/logic/device.py

```diff
@@ -186,490 +196,666 @@ def get_device_display_model(self):
+        """Get the brightness of the light."""
         return self.device_attributes.get("brightness", 0)
 
     def get_color_mode(self):
-        """ Get the color mode of the light. """
+        """Get the color mode of the light."""
         return self.device_attributes.get("color_mode", "unknown")
 
     def get_color_temp(self):
-        """ Get the color temperature of the light. """
+        """Get the color temperature of the light."""
         return self.device_attributes.get("color_temp", 0)
 
     def get_effect(self):
-        """ Get the effect of the light. """
+        """Get the effect of the light."""
         return self.device_attributes.get("effect", "none")
 
     def get_effect_list(self):
-        """ Get the effect list of the light. """
+        """Get the effect list of the light."""
         return self.device_attributes.get("effect_list", [])
 
     def get_hs_color(self):
-        """ Get the hs color of the light. """
+        """Get the hs color of the light."""
         return self.device_attributes.get("hs_color", [0, 0])
 
     def get_max_mireds(self):
-        """ Get the max mireds of the light. """
+        """Get the max mireds of the light."""
         return self.device_attributes.get("max_mireds", 0)
 
     def get_min_mireds(self):
-        """ Get the min mireds of the light. """
+        """Get the min mireds of the light."""
         return self.device_attributes.get("min_mireds", 0)
 
     def get_rgb_color(self):
-        """ Get the rgb color of the light. """
+        """Get the rgb color of the light."""
         return self.device_attributes.get("rgb_color", [0, 0, 0])
 
+    def get_spoken_color(self):
+        """Get the spoken color value of the light."""
+        color = tuple(self.get_rgb_color())
+        try:
+            color = rgb_to_name(color)
+        except ValueError:
+            color = f"RGB code {color[0]}, {color[1]}, {color[2]}"
+        return color
+
     def get_supported_color_modes(self):
-        """ Get the supported color modes of the light. """
+        """Get the supported color modes of the light."""
         return self.device_attributes.get("supported_color_modes", [])
 
     def get_xy_color(self):
-        """ Get the xy color of the light. """
+        """Get the xy color of the light."""
         return self.device_attributes.get("xy_color", [0, 0])
 
     def set_brightness(self, brightness):
-        """ Set the brightness of the light.
+        """Set the brightness of the light.
 
         Args:
             brightness (int): The brightness to set the light to.
         """
-        self.call_function("set_brightness", {"brightness": brightness})
+        LOG.debug(f"Setting brightness to {brightness}")
+        self.call_function("turn_on", {"brightness": brightness})
+        self.update_device()
+
+    def increase_brightness(self) -> int:
+        """Increase the brightness of the light by 10%."""
```

Comment:
Consider making the scaling here configurable

Reply:
Open an issue? There's more complexity there than I'd care to add to an already complex change